bigtable-hbase-2.x-parent/bigtable-hbase-2.x/src/main/java/com/google/cloud/bigtable/hbase2_x/BigtableAsyncAdmin.java

```diff
@@ -750,4 +608,281 @@ public BigtableAsyncAdmin(BigtableAsyncConnection asyncConnection) throws IOExce
       ReplicationPeerConfig arg1) {
     throw new UnsupportedOperationException("updateReplicationPeerConfig");
   }
+
+  @Override
+  public CompletableFuture<Void> addReplicationPeer(String arg0, ReplicationPeerConfig arg1, boolean arg2) {
+    throw new UnsupportedOperationException("updateConfiguration"); // TODO
+  }
+
+  @Override
+  public CompletableFuture<Void> appendReplicationPeerTableCFs(String arg0, Map<TableName, List<String>> arg1) {
+    throw new UnsupportedOperationException("updateConfiguration"); // TODO
+  }
+
+  @Override
+  public CompletableFuture<Boolean> balancerSwitch(boolean arg0) {
+    throw new UnsupportedOperationException("updateConfiguration"); // TODO
+  }
+
+  @Override
+  public CompletableFuture<Boolean> catalogJanitorSwitch(boolean arg0) {
+    throw new UnsupportedOperationException("updateConfiguration"); // TODO
+  }
+
+  @Override
+  public CompletableFuture<Boolean> cleanerChoreSwitch(boolean arg0) {
+    throw new UnsupportedOperationException("updateConfiguration"); // TODO
+  }
+
+  @Override
+  public CompletableFuture<CacheEvictionStats> clearBlockCache(TableName arg0) {
+    throw new UnsupportedOperationException("updateConfiguration"); // TODO
+  }
+
+  @Override
+  public CompletableFuture<Void> compact(TableName arg0, CompactType arg1) {
+    throw new UnsupportedOperationException("updateConfiguration"); // TODO
+  }
+
+  @Override
+  public CompletableFuture<Void> compact(TableName arg0, byte[] arg1, CompactType arg2) {
+    throw new UnsupportedOperationException("updateConfiguration"); // TODO
+  }
+
+  @Override
+  public CompletableFuture<Void> compactRegion(byte[] arg0) {
+    throw new UnsupportedOperationException("updateConfiguration"); // TODO
+  }
+
+  @Override
+  public CompletableFuture<Void> compactRegion(byte[] arg0, byte[] arg1) {
+    throw new UnsupportedOperationException("updateConfiguration"); // TODO
+  }
+
+  @Override
+  public <S, R> CompletableFuture<R> coprocessorService(Function<RpcChannel, S> arg0, ServiceCaller<S, R> arg1) {
+    throw new UnsupportedOperationException("updateConfiguration"); // TODO
+  }
+
+  @Override
+  public <S, R> CompletableFuture<R> coprocessorService(Function<RpcChannel, S> arg0, ServiceCaller<S, R> arg1,
+      ServerName arg2) {
+    throw new UnsupportedOperationException("updateConfiguration"); // TODO
+  }
+
+  @Override
+  public CompletableFuture<Void> createTable(TableDescriptor arg0) {
```

Comment:
This method, and some of the ones below it should be implementable.  Pease add a comment above this method that this should be implemented.

Reply:
Implemented by calling existing createTable and sent empty Optional
  /*
   * (non-Javadoc)
   * @see org.apache.hadoop.hbase.client.AsyncAdmin#createTable(org.apache.hadoop.hbase.client.TableDescriptor)
   */
  @Override
  public CompletableFuture<Void> createTable(TableDescriptor desc) {
    return createTable(desc, Optional.empty());
  }

below method's signature is modified, so changed the scope to private 
private CompletableFuture<Void> createTable(TableDescriptor desc, Optional<byte[][]> splitKeys) 

and new signature method is below
  @Override
  public CompletableFuture<Void> createTable(TableDescriptor desc, byte[][] splitKeys) {
    return createTable(desc, Optional.of(splitKeys));
  }